index.html

```diff
@@ -146,14 +157,25 @@ <h2><a href="#how_to_get_horizontal_review">How to get horizontal review</a></h2
 
 
 <dt>Groups listed in your charter</dt>
-<dd>If there are groups listed in your charter as depending on some specification, you should ask them too.</dd>
+<dd><span class="step">If there are groups listed in your charter as depending on some specification, you should ask them too</span>.</dd>
 
 
 
 <dt> General announcement</dt>
-<dd>Use <a rel="nofollow" class="external text" href="mailto:public-review-announce@w3.org">public-review-announce@w3.org</a> for general announcements regarding wide and horizontal reviews. (Sending an email to this list alone is not sufficient to trigger horizontal reviews. You will still need to formally request review from the Horizontal Groups, as described above.)</dd>
+<dd><span class="step">Use <a rel="nofollow" class="external text" href="mailto:public-review-announce@w3.org">public-review-announce@w3.org</a> for general announcements regarding wide and horizontal reviews. (Sending an email to this list alone is not sufficient to trigger horizontal reviews. You will still need to formally request review from the Horizontal Groups, as described above.)</span></dd>
 </dl>
 
+<details id="githubissue" hidden>
+  <summary>Generate a meta-issue to track horizontal review steps in a GitHub repository</summary>
+  <p>You may find it useful to create an issue in the GitHub repository that hosts a specification that needs to get horizontal reviews to track the different steps involved to request these reviews. Fill the name of your GitHub repository in the field below and hit the "Create GitHub issue" button to open the "new issue" form on the specified GitHub repository, with the issue's body pre-filled with horizontal review steps as a list of checkboxes.</p>
```

Comment:
This is a little difficult to parse.  Here's a suggestion for replacement text:

You may find it useful to create an issue in the GitHub repository of your spec to track your progress. Add the name of your GitHub repository to the field below and hit the "Create GitHub issue" button. This opens the "new issue" form in your repository, and pre-fills the body with horizontal review steps as a list of checkboxes.